spark/templates/spark-webui-proxy-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "webui-proxy-fullname" . }}
+spec:
+  ports:
+    - port: {{ .Values.webUIProxy.service.port }}
+      targetPort: http
+      {{- if .Values.webUIProxy.service.nodePort }}
+      nodePort: {{ .Values.webUIProxy.service.nodePort }}
+      {{- end }}
```

Comment:
Looks kinda weird that only this service supports `nodePort` and the rest do not.

Reply:
You're right! I overlooked it for the others

Reply:
Added them